command/status.go

```diff
@@ -36,12 +36,14 @@ func (c *StatusCommand) Run(args []string) int {
 	}
 
 	outStr := fmt.Sprintf(
-		"Sealed: %v\n"+
+		"Type: %s\n"+
```

Comment:
Looks like there's some weird formatting in here, go fmt to check?

Reply:
It's fmted

Reply:
🤦‍♂️  Yeah my bad, realized that the indentation below is intentional.